library/common/templates/app_functions/_postgres.tpl

```diff
@@ -203,9 +201,7 @@ postgresdata:
         mountPath: /mnt/directories/postgres_data
 postgresbackup:
   enabled: true
-  type: {{ $backup.type }}
-  datasetName: {{ $backup.datasetName | default "" }}
-  hostPath: {{ $backup.hostPath | default "" }}
+  {{- include "ix.v1.common.app.storageOptions" (dict "storage" $backup) | nindent 2 }}
```

Comment:
Just to be sure, `$backup` when used here will always have relevant data properly adjusted already ?

Reply:
Yes! there is a sanity check few lines above to check the dictionary is not empty,
And if there are any required values missing, common chart will throw an error too